features/identity-core/org.wso2.carbon.identity.core.server.feature/resources/org.wso2.carbon.identity.core.server.feature.default.json

```diff
@@ -1230,6 +1230,12 @@
   "console.email_templates.scopes.read": ["internal_email_mgt_view"],
   "console.email_templates.scopes.update": ["internal_email_mgt_update"],
   "console.email_templates.scopes.delete": ["internal_email_mgt_delete"],
+  "console.sms_templates.enabled": false,
+  "console.sms_templates.scopes.feature": ["console:smsTemplates"],
+  "console.sms_templates.scopes.create": ["internal_notification_mgt_create"],
+  "console.sms_templates.scopes.read": ["internal_notification_mgt_view"],
+  "console.sms_templates.scopes.update": ["internal_notification_mgt_update"],
+  "console.sms_templates.scopes.delete": ["internal_notification_mgt_delete"],
```

Comment:
Don't we have dedicated scopes for these?

Reply:
No. Going forward, we have notification template scopes for both email and sms.